src/ansys/dpf/core/label_space.py

```diff
@@ -46,10 +46,22 @@ def _data_processing_core_api(self):
         core_api.init_data_processing_environment(self)
         return core_api
 
-    def fill(self, label_space):
+    def fill(self, label_space: dict):
         for key, index in label_space.items():
             self._api.label_space_add_data(self, key, index)
 
+    def __str__(self):
+        return str(dict(self))
```

Comment:
Is this really good practice?
Maybe you should consider `__repr__` [instead](https://medium.com/@akshatgadodia/understanding-pythons-str-and-repr-methods-a-guide-for-python-developers-83490d83e4e9#:~:text=__str__%20focuses%20on,relevant%20information%20for%20object%20reconstruction.)?

Reply:
@BClappe well, this is __str__, so I'd expect a user-friendly and readable string, such as provided by str().